src/components/RadioGroup/RadioGroup_deprecated.test.js

```diff
@@ -1,9 +1,9 @@
 import React from "react";
-import { render, screen, userEvent, waitFor } from "../utils/test";
+import { render, screen, userEvent, waitFor } from "../../utils/test";
 import "@testing-library/jest-dom/extend-expect";
-import Form from "./Form";
-import RadioGroup from "./RadioGroup";
-import Container from "./Container";
+import Form from "../Form";
+import { RadioGroup } from "..";
```

Comment:
... No, I am importing from `/src/index` which exports all basis components. This will then import the `/src/Radio/index` which will switch between the deprecated and new one.

This ensures that all the tests run against the switch and ensures there are no breaking changes against the existing tests